frontend/src/Pages/StudentProfile.tsx

```diff
@@ -0,0 +1,287 @@
+
+const StudentProfile = () => {
+    const { user_id } = useParams<{ user_id: string }>();
+    const { data, error, isLoading } = useSWR<
+        ServerResponseOne<ResidentEngagementProfile>,
+        AxiosError
+    >(`/api/users/${user_id}/profile`);
+    const metrics = data?.data;
+
+    const isLessAvgThanOneHour =
+        (metrics?.activity_engagement.total_hours_active_weekly ?? 0) < 1;
+    const isLessThanOneHour =
+        (metrics?.activity_engagement.total_hours_engaged ?? 0) < 1;
+
+    const avgNumber = isLessAvgThanOneHour
+        ? (
+              metrics?.activity_engagement.total_minutes_active_weekly ?? 0
+          ).toFixed(2)
+        : (metrics?.activity_engagement.total_hours_active_weekly ?? 0).toFixed(
+              2
+          );
+
+    const weekNumber = isLessThanOneHour ? (metrics?.activity_engagement.total_minutes_engaged ?? 0).toFixed(2) :
+        (metrics?.activity_engagement.total_hours_engaged ?? 0).toFixed(2);
+  
+    const avgLabel = isLessAvgThanOneHour
+        ? 'AVG Minutes PER Week'
+        : 'AVG Hours PER Week';
+
+        const weekLabel = isLessThanOneHour
+        ? 'Total Minutes This Week'
+        : 'Total Hours This Week';
+
+    const avgToolTip = isLessAvgThanOneHour
+        ? 'Average number of minutes resident is logged in to UnlockedEd'
+        : 'Average number of hours resident is logged in to UnlockedEd';
+
+        const weekToolTip = isLessThanOneHour
+        ? 'Total number of minutes resident was logged in to UnlockedEd this week'
+        : 'Total number of hours resident was logged in to UnlockedEd this week';
+
+    return (
+        <div className="overflow-x-hidden px-5 pb-4">
+            {error && <div>Error loading data</div>}
+            {!data || (isLoading && <div>Loading...</div>)}
+            {data && metrics && (
+                <>
+                    <div className="flex flex-row gap-6 items-stretch">
+                        <div className="w-[270px] h-[240px] flex flex-col gap-4">
+                            <div className="card card-row-padding overflow-hidden text-med text-teal-4 flex-1 h-full">
+                                <div className="justify-items-center">
+                                    <UserCircleIcon className="w-[64px] h-[64px]" />
+                                </div>
+                                <div className="">
+                                    <div className="text-md mt-2">
+                                        {
+                                            metrics?.session_engagement
+                                                .user_info.name_first
+                                        }{' '}
+                                        {
+                                            metrics?.session_engagement
+                                                .user_info.name_last
+                                        }
+                                    </div>
```

Comment:
- Can we have the name centered with a larger, possibly bolder font? (see the figma)
- Can we have the fields for username, joined, and last active to mimic the tabular layout in the figma too?